docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+---
+title: ML Models in MindsDB
+---
+
+In this section, we go through the ML model types available in MindsDB. These are regression models, classification models, and time series models.
+
+## What is an ML Model?
+
+A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors and make decisions that lead to predicting future data.
```

Comment:
behaviors ~and make decisions that lead~ to predict future data.

Reply:
Changed:

> A machine learning (ML) model is a program trained using the available data in order to learn how to recognize patterns and behaviors to predict future data.